sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py

```diff
@@ -0,0 +1,2472 @@
+
+        if connection.authentication_type == AuthenticationType.API_KEY:
+            logger.debug(
+                "[InferenceOperations.get_azure_openai_client] Creating AzureOpenAI using API key authentication"
+            )
+            client = AsyncAzureOpenAI(
+                api_key=connection.key, azure_endpoint=connection.endpoint_url, api_version=api_version
+            )
+        elif connection.authentication_type == AuthenticationType.AAD:
+            logger.debug(
+                "[InferenceOperations.get_azure_openai_client] Creating AzureOpenAI using Entra ID authentication"
+            )
+            try:
+                from azure.identity import get_bearer_token_provider
+            except ModuleNotFoundError as _:
+                raise ModuleNotFoundError(
+                    "azure.identity package not installed. Please install it using 'pip install azure-identity'"
+                )
+            client = AsyncAzureOpenAI(
+                # See https://learn.microsoft.com/en-us/python/api/azure-identity/azure.identity?view=azure-python#azure-identity-get-bearer-token-provider
+                azure_ad_token_provider=get_bearer_token_provider(
+                    connection.token_credential, "https://cognitiveservices.azure.com/.default"
+                ),
+                azure_endpoint=connection.endpoint_url,
+                api_version=api_version,
+            )
+        elif connection.authentication_type == AuthenticationType.SAS:
+            logger.debug("[InferenceOperations.get_azure_openai_client] Creating AzureOpenAI using SAS authentication")
+            client = AsyncAzureOpenAI(
+                azure_ad_token_provider=get_bearer_token_provider(
+                    connection.token_credential, "https://cognitiveservices.azure.com/.default"
+                ),
+                azure_endpoint=connection.endpoint_url,
+                api_version=api_version,
+            )
+        else:
+            raise ValueError("Unknown authentication type")
+
+        return client
+
+
+class ConnectionsOperations(ConnectionsOperationsGenerated):
+
+    @distributed_trace_async
+    async def get_default(
+        self, *, connection_type: ConnectionType, with_credentials: bool = False, **kwargs: Any
+    ) -> ConnectionProperties:
+        """Get the properties of the default connection of a certain connection type, with or without
+        populating authentication credentials.
+
+        :param connection_type: The connection type. Required.
+        :type connection_type: ~azure.ai.projects.models._models.ConnectionType
+        :param with_credentials: Whether to populate the connection properties with authentication credentials. Optional.
+        :type with_credentials: bool
+        :return: The connection properties, or `None` if there are no connections of the specified type.
+        :rtype: ~azure.ai.projects.models._models.ConnectionProperties
```

Comment:
Should this be...? We shouldn't be exposing internal models

```suggestion
        :rtype: ~azure.ai.projects.models.ConnectionProperties
```

Reply:
Fixed!